notification-service/src/main/java/org/java/mentorship/notification/service/NotificationService.java

```diff
@@ -0,0 +1,67 @@
+package org.java.mentorship.notification.service;
+
+import org.java.mentorship.notification.domain.NotificationEntity;
+import org.java.mentorship.notification.repository.NotificationChannelRepository;
+import org.java.mentorship.notification.repository.NotificationRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+@Service
+public class NotificationService {
+
+    @Autowired
+    private NotificationRepository notificationRepository;
+
+    @Autowired
+    private NotificationChannelRepository notificationChannelRepository;
+
+    public List<NotificationEntity> getNotifications(Map<String, Object> params) {
+        List<NotificationEntity> notificationEntities = notificationRepository.getNotifications(params);
+        return notificationEntities;
+    }
+
+    public NotificationEntity createNotification(NotificationEntity notification) {
+        NotificationEntity entity = notificationRepository.create(notification);
+
+        // TODO: Get entity ID for creating channels;
+//        notification.getChannels().forEach(channel -> {
+//            notificationChannelRepository.createNotificationChannel(entity.getId(), channel);
+//        });
+
+        return entity;
```

Comment:
I think we can make this method a one-liner

return notificationRepository.create(notification);

Reply:
how can i  modify the create method in repository in order to return the notifcation.id